### PR TITLE
atool recipe

### PR DESCRIPTION
**Pull request by miqlas**
For the emperor!

### CODE REVIEW

**Comment by miqlas** [comment]
'cause i have plenty problems with Expander.

File: app-arch/atool/atool-0.39.0.recipe [hunk 1]
```diff
@@ -0,0 +1,63 @@
+SUMMARY="A script for managing file archives"
+DESCRIPTION="
+The main command is aunpack which extracts files from an archive. \
+Did you ever extract files from an archive, not checking whether the \
+files were located in a subdirectory or in the top directory of the archive, \
+resulting in files scattered all over the place? aunpack overcomes this problem \
+by first extracting to a new directory. If there was only a single file in the archive, \
+that file is moved to the original directory. aunpack also prevents local files from \
+being overwritten by mistake. \
```
Comment (diversys) [inline app-arch/atool/atool-0.39.0.recipe]: You don't need backslash here and on the next line

File: app-arch/atool/atool-0.39.0.recipe [hunk 2]
```diff
@@ -0,0 +1,63 @@
+SUMMARY="A script for managing file archives"
+DESCRIPTION="
+The main command is aunpack which extracts files from an archive. \
```
Comment (diversys) [inline app-arch/atool/atool-0.39.0.recipe]: Start form the previous line

File: app-arch/atool/atool-0.39.0.recipe [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+archives), and acat (to extract files to standard out). As atool invokes external \
+programs to handle the archives, not all commands may be supported for a \
+certain type of archives.
+
+atool identifies archives by their file extension. Sometimes this is not possible \
+- for instance rar archives usually have varying numeric file extensions. In those \
+cases when atool can't identify the format, file is used instead. (atool can be \
+configured not to use file.)"
+HOMEPAGE="http://www.nongnu.org/atool/"
+COPYRIGHT="2001-2011 Oskar Liljeblad."
```
Comment (diversys) [inline app-arch/atool/atool-0.39.0.recipe]: Period at the end is superfluous

File: app-arch/atool/atool-0.39.0.recipe [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+The other commands provided are apack (to create archives), als (to list files in \
+archives), and acat (to extract files to standard out). As atool invokes external \
+programs to handle the archives, not all commands may be supported for a \
+certain type of archives.
+
+atool identifies archives by their file extension. Sometimes this is not possible \
+- for instance rar archives usually have varying numeric file extensions. In those \
+cases when atool can't identify the format, file is used instead. (atool can be \
+configured not to use file.)"
```
Comment (korli) [inline app-arch/atool/atool-0.39.0.recipe]: Does atool required file?

File: app-arch/atool/atool-0.39.0.recipe [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+HOMEPAGE="http://www.nongnu.org/atool/"
+COPYRIGHT="2001-2011 Oskar Liljeblad"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://savannah.nongnu.org/download/atool/atool-$portVersion.tar.gz"
+CHECKSUM_SHA256="aaf60095884abb872e25f8e919a8a63d0dabaeca46faeba87d12812d6efc703b"
+SOURCE_DIR="atool-$portVersion"
```
Comment (korli) [inline app-arch/atool/atool-0.39.0.recipe]: I believe SOURCE_DIR is defined to the default value, thus can be removed.

File: app-arch/atool/atool-0.39.0.recipe [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+	cmd:adiff = $portVersion
+	cmd:als = $portVersion
+	cmd:apack = $portVersion
+	cmd:arepack = $portVersion
+	cmd:aunpack = $portVersion
+	"
+REQUIRES="
+	haiku
+	cmd:perl
+	cmd:awk
```
Comment (korli) [inline app-arch/atool/atool-0.39.0.recipe]: Why would it need awk?

**Comment by waddlesplash** [comment]
@miqlas bump?

File: app-arch/atool/atool-0.39.0.recipe (hunk 4 above)
Comment (miqlas) [inline app-arch/atool/atool-0.39.0.recipe]: Yep.

Comment (waddlesplash) [inline app-arch/atool/atool-0.39.0.recipe]: It says it can be configured to not use file, so I'm assuming it's not a hard dependency.

**Pull request merged**